simple/src/BankCustomerData.java

```diff
@@ -37,12 +37,14 @@ public BankCustomerData(HashMap<String, Customer> nameToCustomer,
      * Will not add.
      */
     public void addCustomer(Customer customer) {
-        if (!containsCustomer(Customer.getHashName(customer))) {
+        // TODO(Eddie): Remove this.
+        if (containsCustomer(customer.getFullName())) {
             // TODO(Alex): Should we throw here if we try to add a customer
```

Comment:
The responsibility of this method is not to print anything to the user so I will add the throw exception with the error message which the menu should handle instead of this function.

Reply:
sg